GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -177,6 +178,17 @@ private async void btnDownloadNow_Click(object sender, EventArgs e)
             {
             }
         }
+
+        private void linkDirectDownload_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            try
+            {
+                Process.Start(UpdateUrl);
+            }
+            catch (Win32Exception)
+            {
+            }
```

Comment:
* What sort of exceptions are we expecting?
* By swallowing exceptions we're creating a new problem - something isn't working and we don't know the cause. It is better to throw and let the NBug handler create a report.

Reply:
I just restored the original code which didn't have anything. I'm not entirely sure what exceptions we would expect when we tell the OS to launch a URL via the Default browser.

Rather than catch a Win32 exception, perhaps we catch all exceptions and we can send it via NBug. Are you happy with this approach?

I haven't used NBug but i assume it is similar to Microsoft AppCentre.

Reply:
No need to `try..catch` - an exception will be trapped by NBug via global handlers

Reply:
Great, I'll remove the try..catch in that case